analysis/query.sql

```diff
@@ -1,3 +1,5 @@
+-- If you change the values of these variables,
```

Comment:
It could! Such an action could also use Jinja2, as the `render_report` action does. That said, whilst I wouldn't say the current implementation is beautiful, I also wouldn't say it needs further ornamentation.

To put it another way, I don't think the risks justify the effort. And if they did, or came close, then the effort should probably be invested in SQL Runner rather than in this study (opensafely-core/sqlrunner#72).

Reply:
that would be a much more sensible approach!